app/src/main/java/org/luxoft/sdl_core/MainActivity.java

```diff
@@ -41,6 +44,10 @@
     private Button stop_sdl_button;
     public static String sdl_cache_folder_path;
     public static String sdl_external_dir_folder_path;
+    private static final int ACCESS_LOCATION_REQUEST = 1;
+    private static final int ACCESS_EXT_STORAGE_REQUEST = 2;
+    public final static String ACTION_START_BLE = "ACTION_START_BLE";
+    public final static String ACTION_STOP_BLE = "ACTION_STOP_BLE";
```

Comment:
@ychernysheva actions related to service should be declared in the corresponding service. Please move these two constants to `BleCentralService`

Reply:
Fixed in be07980